### PR TITLE
rest: adds a possibility to query logs for a given step

### PR DESCRIPTION
**Pull request by roksys**
* Addresses reanahub/reana-client/issues/369

### CODE REVIEW

File: docs/openapi.json [hunk 1]
```diff
@@ -1349,6 +1349,20 @@
...
             "required": true,
             "type": "string"
+          },
+          {
+            "description": "Steps of a workflow.",
+            "in": "body",
+            "name": "steps",
+            "required": false,
+            "schema": {
+              "description": "List of steps names to get logs.",
```
Comment (tiborsimko) [inline docs/openapi.json]: (see my other cosmetic comment WRT phrasing)

**Comment by diegodelemos** [comment]
There is a test not passing, `test_get_workflow_logs`.

**Comment by roksys** [comment]
It's failing because openapi specs were changed. With the new REANA-commons should be fine.

**Comment by diegodelemos** [comment]
You can bump R-Commons to `0.7.0.dev20200211` :)

**Pull request merged**